azerothcore/azerothcore-wotlk
# Handle custom creatures IDs and exclude them from checks

## Changes Proposed:
-  Created list of custom Creature IDs in worldserver.conf. These IDs will be skipped from some definitions checks

## Tests Performed:

- If you won't to correctly add records for every custom NPC to `gossip_menu` tables - just put all these IDs in this list and server will not distrub you by warnings.
- How @Kitzunu sad, "I dont want to be the one to fix 100 modules. But  the exception is easy". 


## How to Test the Changes:


1. Add your custom IDs to worldserver.conf list `Creatures.CustomIDs` and restart the server.  
2. There should be no messages in the console about those Creatures.
3.


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
